src/explore-education-statistics-common/src/modules/table-tool/components/FormSortableList.module.scss

```diff
@@ -26,47 +35,60 @@
   }
 
   &:focus {
-    border: $govuk-focus-width solid govuk-colour('black');
     outline: $govuk-focus-colour solid $govuk-focus-width;
+    outline-offset: $govuk-focus-width * -1;
   }
 }
 
-.optionText {
-  display: flex;
-  justify-content: space-between;
-}
+.isSelected {
+  background: govuk-colour('blue');
+  color: govuk-colour('white');
 
-.optionActions {
-  display: flex;
-  flex-direction: column;
+  &:hover {
+    background: darken(govuk-colour('blue'), 5);
+  }
 }
 
-.optionDragging {
+.isDragging {
   background: govuk-colour('blue');
   border: $govuk-focus-width solid govuk-colour('black');
```

Comment:
Just noticed that when dragging, the border causes displacement of the child elements inside of the box. Think we can fix this with:

```scss
box-shadow: inset 0 0 0 $govuk-focus-width govuk-colour('black');
```